openslides/motions/models.py

```diff
@@ -758,7 +758,7 @@ class MotionChangeRecommendation(RESTModelMixin, models.Model):
     """The motion version to which the change recommendation belongs."""
 
     status = models.PositiveIntegerField(default=0)
```

Comment:
I thought we wanted to use a boolean field (rejected false/true). Status 0 should not means accepted because the change recomendation is not “accepted” by default.